bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/State.java

```diff
@@ -184,4 +189,51 @@ public Effect getEffect() {
     public boolean isReachable() {
         return reachable;
     }
+
+    /*
+     * (non-Javadoc)
+     * 
+     * @see java.lang.Object#equals(java.lang.Object)
+     */
```

Comment:
This javadoc isn't adding any new information, so please remove it.

Reply:
Removed

Reply:
Removed it